src/style.css

```diff
@@ -0,0 +1,21 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.box {
+  width: 100%;
+  height: 100px;
+}
+
+#red {
```

Comment:
do not use id's for styling, add second class to blocks instead
`class="box red"`